hardhat.config.js

```diff
@@ -0,0 +1,164 @@
+require('dotenv').config()
+
+require('@1hive/hardhat-aragon')
+require('@nomiclabs/hardhat-ethers')
+require('@nomiclabs/hardhat-truffle5')
+require('@nomiclabs/hardhat-web3')
+require('hardhat-deploy')
+require('hardhat-gas-reporter')
+require('solidity-coverage')
+
+const { node_url, accounts } = require('./utils/network')
+
+process.removeAllListeners('warning')
+module.exports = {
+  solidity: {
+    compilers: [
+      {
+        version: '0.4.24',
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 10000,
+          },
+        },
+      },
+    ],
+  },
+  aragon: {
```

Comment:
The `aragon` config extension replaces the `arapp.json` file. To have more context check README of [`@1hive/hardhat-aragon` plugin](https://github.com/1Hive/hardhat-aragon/blob/master/README.md#config-extensions).